cmd/bbolt/command_surgery_cobra.go

```diff
@@ -134,12 +155,36 @@ func surgeryCopyPageFunc(srcDBPath string, cfg surgeryOptions) error {
 		fmt.Fprintf(os.Stdout, "Please consider executing `./bbolt surgery abandon-freelist ...`\n")
 	}
 
-	fmt.Fprintf(os.Stdout, "The page %d was successfully copied to page %d\n", cfg.surgerySourcePageId, cfg.surgeryDestinationPageId)
+	fmt.Fprintf(os.Stdout, "The page %d was successfully copied to page %d\n", cfg.sourcePageId, cfg.destinationPageId)
+	return nil
+}
+
+type clearPageElementsOptions struct {
+	surgeryBaseOptions
+	pageId         uint64
+	startElementId int
+	endElementId   int
```

Comment:
nit: `startElementIdx` or `startElementIndex` is better than `startElementId`. There is no a concept `element ID`, and it isn't global unique either.
```suggestion
	startElementIdx int
	endElementIdx   int
```

Reply:
Done